collect_app/src/androidTest/java/org/odk/collect/android/feature/formentry/FormMediaTest.kt

```diff
@@ -5,26 +5,61 @@ import org.junit.Rule
 import org.junit.Test
 import org.junit.rules.RuleChain
 import org.junit.runner.RunWith
-import org.odk.collect.android.support.rules.FormEntryActivityTestRule
+import org.odk.collect.android.storage.StoragePathProvider
+import org.odk.collect.android.storage.StorageSubdirectory
+import org.odk.collect.android.support.rules.CollectTestRule
 import org.odk.collect.android.support.rules.TestRuleChain.chain
 
 @RunWith(AndroidJUnit4::class)
 class FormMediaTest {
 
-    private val rule = FormEntryActivityTestRule()
+    private val rule = CollectTestRule()
 
     @get:Rule
-    val ruleChain: RuleChain = chain()
-        .around(rule)
+    val ruleChain: RuleChain = chain().around(rule)
 
     @Test
     fun loadingFormWithZippedMedia_unzipsIntoMediaDirectory() {
-        rule.setUpProjectAndCopyForm("external_select_10.xml", listOf("external_data_10.zip"))
-            .fillNewForm("external_select_10.xml", "external select 10")
+        rule.startAtMainMenu()
+            .copyForm("external_select_10.xml", listOf("external_data_10.zip"))
+            .startBlankForm("external select 10")
             .clickOnText("a")
             .swipeToNextQuestion("Second")
             .assertText("aa")
             .assertText("ab")
             .assertText("ac")
     }
+
+    @Test
+    fun missingFileMessage_shouldBeDisplayedIfExternalFileIsMissing() {
+        val formsDirPath = StoragePathProvider().getOdkDirPath(StorageSubdirectory.FORMS)
+
+        rule.startAtMainMenu()
+            .copyForm("search_and_select.xml")
```

Comment:
I think we could split these out into their relevant select tests (`ExternalSelectsTest`, `ExternalSecondaryInstanceTest` etc)

Reply:
Done.